CHANGELOG.md

```diff
@@ -760,6 +768,7 @@ Version 2.0 of the UI framework is built for player 7.1. If absolutely necessary
 ## 1.0.0 (2017-02-03)
 - First release
 
+[3.33.0]: https://github.com/bitmovin/bitmovin-player-ui/compare/v3.32.0...v3.33.0
```

Comment:
Don't we usually also include a `develop` entry to compare the current release to `develop`?

Reply:
Yes, but not in this PR as we are merging to `main` here 👍